src/models/converters/mod.rs

```diff
@@ -18,3 +20,15 @@ pub(super) fn get_transfer_direction(safe: &str, from: &str, to: &str) -> Transf
         TransferDirection::Unknown
     }
 }
+
+pub(super) fn get_address_info(
+    safe: &str,
+    address: &str,
+    info_provide: &mut dyn InfoProvider,
+) -> Option<AddressInfo> {
+    if safe != address {
```

Comment:
Could we add a comment for this? 

Just to avoid that we look at this in 1 week and have no idea why we did this.